tron/bin/check_tron_jobs.py

```diff
@@ -115,23 +126,20 @@ def compute_check_result_for_job_runs(client, job, job_content):
         prefix = "UNKNOWN: The job is in a state that check_tron_jobs doesn't understand"
         status = 3
 
+    precious_runs_note = ''
+    if _are_runs_precious(job['monitoring']):
+        precious_runs_note = f"Note: Job {job_content['name']} is precious. Owners will be alerted if a day's, instead of the overall job's, latest run failed.\n"
```

Comment:
`Note: This alert is the run for 2018-10-04 (Friday). A resolve event will not occur until a job run for this date succeeds.`

How are we going to let people "skip" these?....